addons/default/visiosoft/advs-module/resources/views/advs/list-item.twig

```diff
@@ -270,13 +270,18 @@
                                                 {% for feature in  features %}
                                                     <li>
                                                         <h5>{{ feature.name ? feature.name : '' }} </h5>
-
-                                                        {% set array = feature.custom_field_value|split(',') %}
-                                                        <ul>
-                                                            {% for value in array %}
-                                                                <li><i class="fa fa-check-circle"></i>{{ value }}</li>
-                                                            {% endfor %}
-                                                        </ul>
+                                                        {% if feature.custom_field_value is iterable %}
+                                                            {% set array = feature.custom_field_value %}
+                                                            <ul>
+                                                                {% for value in array %}
```

Comment:
I'm getting this for fix right now but Can we move custom field view code to it's own repo? @onurure